site-manage/datemanager/impl/src/test/java/org/sakaiproject/datemanager/test/DateManagerServiceTest.java

```diff
@@ -0,0 +1,153 @@
+package org.sakaiproject.datemanager.test;
+
+import lombok.extern.slf4j.Slf4j;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONValue;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.sakaiproject.datemanager.api.DateManagerConstants;
+import org.sakaiproject.datemanager.api.DateManagerService;
+import org.sakaiproject.datemanager.api.model.DateManagerUpdate;
+import org.sakaiproject.datemanager.api.model.DateManagerValidation;
+import org.sakaiproject.grading.api.Assignment;
+import org.sakaiproject.grading.api.GradingService;
+import org.sakaiproject.time.api.UserTimeService;
+import org.sakaiproject.tool.api.SessionManager;
+import org.sakaiproject.tool.api.Tool;
+import org.sakaiproject.tool.api.ToolManager;
+import org.sakaiproject.tool.api.ToolSession;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.util.FileCopyUtils;
+
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.Date;
+import java.util.TimeZone;
+import java.util.UUID;
+
+import static org.mockito.Mockito.when;
+
+@Slf4j
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = {DateManagerTestConfiguration.class})
+public class DateManagerServiceTest {
```

Comment:
Why not use the SakaiTest from kernel. along with the SakaiTestConfiguration? Saves some boilerplate

Reply:
good call, thx